src/App.tsx

```diff
@@ -7,8 +7,8 @@ import Input from './components/Input/Input';
 import RepoCard from './components/RepoCard/RepoCard';
 import Tab from './components/Tab/Tab';
 import { GithubRepo, fetchGithubRepos } from './GithubAPI';
-import IssuesList from './components/IssuesList';
-import CommitsList from './components/CommitsList';
+import IssuesList from './components/Issue and Commits List/IssuesList';
```

Comment:
Never ever have a space in a directory! You can run on big problems depending on the OS